rootfs/defaults/jicofo.conf

```diff
@@ -0,0 +1,159 @@
+{{ $ENABLE_AUTH := .Env.ENABLE_AUTH | default "0" | toBool }}
```

Comment:
This is helm template

Reply:
https://github.com/talview/jicofo/blob/65d0a3d612d59052af8c467a44818bf4742fd9c4/rootfs/etc/cont-init.d/10-config#L17